pandas/core/generic.py

```diff
@@ -7784,25 +7784,49 @@ def _doc_parms(cls):
 %(outname)s : %(name1)s or %(name2)s (if level specified)\n"""
 
 _bool_doc = """
-
 %(desc)s
+Returns True if one (or more) elements are non-zero,
+not-empty or not-False.
+
+Also note that a series consisting of different
+data types returns the first occurence of the
+non-zero, not-empty or not-False element.
 
 Parameters
 ----------
 axis : %(axis_descr)s
 skipna : boolean, default True
     Exclude NA/null values. If an entire row/column is NA, the result
-    will be NA
+    will be NA.
 level : int or level name, default None
     If the axis is a MultiIndex (hierarchical), count along a
-    particular level, collapsing into a %(name1)s
+    particular level, collapsing into a %(name1)s.
 bool_only : boolean, default None
     Include only boolean columns. If None, will attempt to use everything,
     then use only boolean data. Not implemented for Series.
+**kwargs :
+    Additional keywords have no effect but might be accepted for
+    compatibility with numpy.
 
 Returns
 -------
-%(outname)s : %(name1)s or %(name2)s (if level specified)\n"""
+%(outname)s : %(name1)s or %(name2)s (if level specified)
+
+Examples
+--------
+>>> s1 = pd.Series([1,2,3])
```

Comment:
Could you add spaces after the commas here so this is pep8 compliant?

Reply:
Okay, will do.

Reply:
Done.